services/talent-agent/schemas.py

```diff
@@ -0,0 +1,583 @@
+from kor.nodes import Object, Text, Number
+
+personal_info_schema = Object(
+    id="personal_info",
+    description="Personal information about a person.",
+    attributes=[
+        Text(
+            id="full_name",
+            description="The full name of the candidate.",
+        ),
+        Text(
+            id="email",
+            description="The email address of the candidate.",
+        ),
+        Text(
+            id="phone",
+            description="The contact number of the candidate.",
+        ),
+        Text(
+            id="address",
+            description="The physical address of the candidate without the country.",
+        ),
+        Text(
+            id="country",
+            description="The country of residence of the candidate.",
+        ),
+    ],
+    examples=[
+        (
+            (
+                "ASHLEY GILL 3 Lappage Court Telephone: 01882 652349 Tyler Green, Bucks. Mobile: 07717 121824 HP8 4JD "
+                "Email: ashleygill2023@gotmail.com"
+            ),
+            [
+                {
+                    "full_name": "Ashley Gill",
+                    "email": "ashleygill2023@gotmail.com",
+                    "phone": "01882 652349",
+                    "address": "3 Lappage Court Tyler Green, Bucks.",
+                    "country": "",
+                }
+            ],
+        ),
+        (
+            (
+                "Hasanga University of Moratuwa, Sri Lanka 2021 - Present Ranasinghe B.Sc. Engineering (Hons) Specialised "
+                "in Computer Science and Engineering Computer Science & Stream: Data Science and Engineering Combined "
+                "Mathematics(A), Physics(A), Chemistry(A) +94 70 168 4781 Z-Score: 2.0883 G.C.E Ordinary Level Examination "
+                "(2017) hasanga-ranasinghe 8 A’s hasanga1 dinithhasangare@gmail.com Projects Kadawatha, Sri Lanka."
+            ),
+            [
+                {
+                    "full_name": "Hasanga Ranasinghe",
```

Comment:
Shall we remove the PII (Personally Identifiable Information) and use dummy data?